docs/release/trg-0/trg-9-02.md

```diff
@@ -0,0 +1,154 @@
+---
+title: TRG 9.02 - KIT Maturity Levels
+---
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft      | 119-Feb-2024  | Initial contribution                   |
+
+## KIT Maturity Levels
+
+KITs have a maturity level of sandbox, incubating, or graduated.
+
+The maturity levels of KITs signals the quality, completeness and proof that the use case solves specific business problem and provides value to all stakeholders.
+
+In order to be promoted from one level to the next, the developers must fulfill various criteria. For each stage, different artifacts and deliverables are defined that must be completed. A distinction is made between mandatory (✓) and recommended ((✓)) deliverables. In order to be promoted to the next level, a kit must have completed all necessary items of the lower level and at least one necessary item of the following level. As soon as this criteria is met a promotion has to be requested. To achieve graduated status, additionally a case study must be completed with a partner from the community. The aim of this case study is to verify the content of the KIT. The performing partner tries to solve the problem only with the help of the KIT and, if necessary, to make suggestions for improvement that have to be implemented.
+
+## Initiating a new KIT
+
+To start a Kit or to be in Sandbox phase you could also be in one of two categories.
+
+Option 1: High maturity. The reason to establish a Kit is clear, you have a scope and other interested companies in contributing to that KIT. Please go directly to Sandbox or Incubating stage.
+
+Option 2: You have an idea and want to see if there is any relevance for this KIT. You still need to find other interested companies for contributing to this KIT:
+
+1. Create a discussion in the SIG-Release repository under the "Ideas" category [SIG-Release repository - discussions](https://github.com/eclipse-tractusx/sig-release/discussions/categories/ideas ).
+2. Provide information on the industry problem that this KIT will solve and the domain it should be placed in.
+3. Seek feedback from the community and find other supporters (recommended four companies).
+4. Wait for approval from corresponding committer, who will review the repository and new KIT requests./
+Create request for new repository
+
+Once approved, you can start with the Sandbox or move directly into Incubating.
+
+## Sandbox Stage
+
+Sandbox phase: Definition
```

Comment:
this sentence doesn't feel like its adding value

Reply:
Deleted.